spec/presenters/vendor_api/v1.3/application_presenter_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe VendorAPI::ApplicationPresenter do
+  let(:version) { '1.3' }
+  let(:application_json) { described_class.new(version, application_choice).as_json }
+  let(:attributes) { application_json[:attributes] }
+  let(:application_form) { application_choice.application_form }
+
+  describe '#references' do
+    let(:reference) do
+      build(:reference, reference_status)
+    end
+
+    let(:reference_status) { :feedback_provided }
+
+    before do
+      application_form.application_references.destroy_all
```

Comment:
might be useful to have `:without_references` as an application choice factory trait